user_management/doc/User_management.md

```diff
@@ -180,3 +180,30 @@ all fields are mandatory
 > - An unexpected error occurred
 </details>
 
+
+## `user/{user_id}`
+### Get user non-sensitive information
+
+will return a user object when successful
+Might be extended to return more information in the future, if needed
+
+<details>
+ <summary><code>GET</code><code><b>/user/{user_id}/</b></code></summary>
+
+### Parameters
+
+#### In the URL (mandatory)
+ {user_id}
+> 
+> NB : user_id must be an integer
+> 
```

Comment:
remove this, it's trivial

Reply:
I think this is fine. It might seem obvious, but at least the documentation is clear

Reply:
This is not trivial if the route accepts strings instead of integers